### ISSUE TITLE
[Bug] Logs for Message edited by author doesn't work.

### ISSUE THREAD

**Issue by JjTheHusky** [GilbertGobbels/GAwesomeBot#30]
This has been an issue with BitQuotes files as well, but the bot will not log Messages edited by author, only messages deleted.

**Comment by KingDGrizzle** [comment]
Have you enabled the logging option under Moderation/Status Messases?

**Comment by JjTheHusky** [comment]
Yes, I have set it with the same settings and channel as the message delete.
![image](https://cloud.githubusercontent.com/assets/25309518/22226645/e771bb9e-e18c-11e6-809c-bab834059c27.png)

**Comment by KingDGrizzle** [comment]
`const targetChannelDocument = serverDocument.channels.id(ch.id);
								if(!targetChannelDocument || targetChannelDocument.bot_enabled) {
									ch.createMessage(`Message by **@${bot.getName(msg.guild, serverDocument, msg.member)}** in #${msg.channel.name} edited. Original:\n\`\`\`${oldmsgdata.cleanContent}\`\`\`Updated:\n\`\`\`${msg.cleanContent}\`\`\``, {disable_everyone: true});
								}` This seems to fail. Will look into it localy, and see if I can PR a fix

**Comment by KingDGrizzle** [comment]
AHA. Speak of the devil in #31, we got the same issue in messageUpdate

**Comment by KingDGrizzle** [comment]
Actually, thats not necessarily the case. Seems like something just doesn't trigger

**Comment by KingDGrizzle** [comment]
Reading through the Eris wiki, it seems like we have to cache the old message, which I don't know how, waiting for a reply from DAPI/eris

**Comment by KingDGrizzle** [comment]
**INFO:** Its not the current bot that's causing the issue, but I'm looking in the issue and if its fixable
